ui/pages/SccsPage.qml

```diff
@@ -43,7 +43,7 @@ CoreLayout {
 
                 text: qsTr("Continuous Deployment")
 
-                //enabled: root.repositoryName !== "" && root.environment !== ""
+                enabled: root.repositoryName !== "" && root.environment !== ""
```

Comment:
Regression

This is normal to have it commented. You don't need to select an environment to trigger the continuous deployment view. This line should be removed in fact.